SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -606,19 +607,25 @@ void ASpatialDebugger::DrawDebug(UCanvas* Canvas, APlayerController* /* Controll
 	}
 #endif
 
+	if (bSelectActor)
```

Comment:
I don't see bSelectActor set to true anywhere

Reply:
Oh - it's set in blueprints?

Reply:
Yes it's set by the user via the Blueprints: WBP_SpatialDebuggerConfigMenu -> added checkbox Select Actor and bound to SpatialDebugger::bSelectActor.